outputs.tf

```diff
@@ -1,9 +1,49 @@
 output "sns_topic" {
-  value       = aws_sns_topic.this
+  value       = local.enabled ? aws_sns_topic.this : null
```

Comment:
`aws_sns_topic.this` will be a list with one item (or zero, if enabled=false).
Is it what we want?